File: Lista3 - py/Exercicios-Sala/Exercicio9.py
B = int(input("Digite sua base: "))
EI = int(input("Digite o valor inicial do seu expoente: "))
EF = int(input("Digite o valor final do seu expoente: "))
for x in range(EI, EF+1, 1):
    t = B ** x
    print(f"{B} elevado a {x} = ",t)

File: Lista3 - C/ExerciciosSala/Exercicio9.c
#include <stdio.h>
#include <math.h>
int main() {
    int B, EI, EF;
    printf("Digite sua base: ");
    scanf("%d", &B);
    printf("Digite o valor inicial do seu expoente: ");
    scanf("%d", &EI);
    printf("Digite o valor final do seu expoente: ");
    scanf("%d", &EF);
    for (int x = EI; x <= EF; x++) {

        int t = (int)pow(B, x);
        printf("%d elevado a %d = %d\n", B, x, t);
    }
}
